src/util/pointerSupport.js

```diff
@@ -0,0 +1,29 @@
+import { addOnceEventListener } from './helpers'
+
+/**
+ * @mixin
+ */
+export default {
+  mounted () {
+    const setupPointerSupport = (propName, eventName) => {
+      const load = () => {
+        const app = document.querySelector('[data-app]') || document.body
+        const className = 'application--' + propName.replace(/([A-Z])/g, '-$1').toLowerCase()
+        app.classList.add(className)
+      }
+
+      addOnceEventListener(window, eventName, () => {
+        this.$vuetify[propName] = true
+        this.$vuetify.load(load)
```

Comment:
Why wait for load? If the v-app has mounted then it's ready to go, no reason to wait for images.